cdc/model/sink.go

```diff
@@ -85,9 +92,10 @@ type RowChangedEvent struct {
 
 // Column represents a column value in row changed event
 type Column struct {
-	Type        byte        `json:"t"`
-	WhereHandle *bool       `json:"h,omitempty"`
-	Value       interface{} `json:"v"`
+	Type        byte         `json:"t"`
+	WhereHandle *bool        `json:"h,omitempty"`
+	Flag        bitflag.Flag `json:"f"`
```

Comment:
could you add the unit test for this bitflag?

Reply:
OK, I will implement it at pkg/util/bitflag.go, and use pingcap check to write some uint tests.

Reply:
and I found that the `bitflag.Flag` is alias of `byte`, so the `bitflag.Flag` can store up to 8 bits of flags. It seems that 8 bits are not enough, could we using `uint64` to store the flags?

Reply:
Sure!